src/sidebar/util/dom.js

```diff
@@ -17,7 +18,7 @@ export function getElementHeightWithMargins(element) {
   const marginHeight =
     parseFloat(style.marginTop) + parseFloat(style.marginBottom);
 
-  return elementHeight + marginHeight;
+  return Math.ceil(elementHeight + marginHeight);
```

Comment:
Can you remove this for the moment please. I'm not certain whether we want to do this as it may introduce a minor discrepancy between the amount of space used when an element is rendered vs when it is not rendered - depending on whether the browser snaps elements to the nearest pixel on layout, and I'm sure that the infinite scroll was caused by something else.